core/src/epicli/tests/helpers/test_data_loader.py

```diff
@@ -0,0 +1,95 @@
+                                                      'secret': 'XXXXXXXXXXXXXXXX'
+                                                  },
+                                                  'default_os_image': 'default'},
+                                                 'components':
+                                                 {'repository': {'count': 1},
+                                                  'kubernetes_master': {'count': 1},
+                                                  'kubernetes_node': {'count': 2},
+                                                  'logging': {'count': 1},
+                                                  'monitoring': {'count': 1},
+                                                  'kafka': {'count': 2},
+                                                  'postgresql': {'count': 1},
+                                                  'load_balancer': {'count': 1},
+                                                  'rabbitmq': {'count': 1}}}}
+
+
+def test_get_data_dir_path():
+    assert get_data_dir_path() == os.path.realpath(
+        os.path.join(SCRIPT_DIR, DATA_FOLDER_PATH))
+
+
+def test_get_provider_subdir_path():
+    assert get_provider_subdir_path("terraform", "aws") == os.path.realpath(
+        os.path.join(SCRIPT_DIR, DATA_FOLDER_PATH, "aws", "terraform"))
+
+
+def test_load_manifest_docs():
+    build_path = get_build_path(CLUSTER_NAME_LOAD)
+    docs = load_manifest_docs(build_path)
+    assert docs == TEST_DOCS
+
+
+def test_load_not_existing_manifest_docs():
+    build_path = get_build_path(NON_EXISTING_CLUSTER)
+    with pytest.raises(Exception):
+        load_manifest_docs(build_path)
+
+
+def test_load_json_obj():
+    loaded_json = load_json_obj(os.path.join(OUTPUT_PATH, TEST_JSON_NAME))
+    assert loaded_json == TEST_JSON
+
+
+def test_load_template_file():
+    template = load_template_file(types.ANSIBLE, "common", "ansible_inventory")
+    content = template.render(inventory=TEST_INVENTORY,
+                              cluster_model=TEST_CLUSTER_MODEL)
+    assert 'test-1 ansible_host=10.0.0.1' in content
+    assert 'test-2 ansible_host=10.0.0.2' in content
+    assert 'test-3 ansible_host=10.0.0.3' in content
+    assert 'test-4 ansible_host=10.0.0.4' in content
+    assert 'ansible_user=operations' in content
+    assert 'ansible_ssh_private_key_file=id_rsa' in content
+
+
+def test_load_all_yaml_objs():
+    yaml_objs = load_all_yaml_objs(
+        types.DEFAULT, "aws", 'configuration/minimal-cluster-config')
+    assert yaml_objs == [TEST_MINIMAL_CLUSTER_CONFIG]
+
+
+def test_load_yaml_obj():
+    yaml_obj = load_yaml_obj(types.DEFAULT, 'aws',
+                             'configuration/minimal-cluster-config')
+    assert yaml_obj == TEST_MINIMAL_CLUSTER_CONFIG
+
+
+def test_load_file_from_path():
+    path_to_file = os.path.realpath(
+        os.path.join(SCRIPT_DIR, DATA_FOLDER_PATH, 'aws', types.DEFAULT, 'configuration/minimal-cluster-config' + '.yml'))
```

Comment:
Use f-strings

Reply:
Instead of f-string maybe just remove the +?